src/main/java/by/itechart/api/controller/UserControllerInfo.java

```diff
@@ -3,16 +3,33 @@
 import by.itechart.api.dto.CreateUserDTO;
 import by.itechart.api.dto.UpdateUserDTO;
 import by.itechart.api.dto.UserDTO;
+import by.itechart.api.util.annotation.ApiPageable;
 import io.swagger.annotations.*;
 import io.swagger.v3.oas.annotations.Operation;
 import io.swagger.v3.oas.annotations.security.SecurityRequirement;
 import org.springframework.data.domain.Pageable;
 import org.springframework.http.ResponseEntity;
 import org.springframework.security.core.Authentication;
+import springfox.documentation.annotations.ApiIgnore;
 
 import java.util.List;
 
 public interface UserControllerInfo {
+
+    @Operation(security = @SecurityRequirement(name = "basicAuth"))
+    @ApiOperation(value = "Get all active users", notes = "This method will return all active users", authorizations = {
+            @Authorization(value = "basic", scopes = {@AuthorizationScope(scope = "getUsers", description = "getAllUsers")})
+    })
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "Successfully returned all active users"),
+            @ApiResponse(code = 401, message = "Unauthorized to perform this operation"),
+            @ApiResponse(code = 403, message = "Forbidden access to resource"),
+            @ApiResponse(code = 500, message = "Server error. Something went wrong")})
+    @ApiPageable
+    ResponseEntity<List<UserDTO>> getActiveUsers(@ApiParam(value = "Pagination information which can be defined with page, page size and sort") @ApiIgnore Pageable pageable,
```

Comment:
It seems to me you are making some contradictory things. 
You added `@ApiIgnore`, does it make sense to add a description for this param in `@ApiParam`?
Another question is whether it's possible to apply `@ApiPageable` directly to the parameter instead of the method?

Reply:
> You added @ApiIgnore, does it make sense to add a description for this param in @ApiParam?

Yes, because `@ApiIgnore` will hide unused statements in swagger for pageable.

> Another question is whether it's possible to apply @ApiPageable directly to the parameter instead of the method?

Unfortunately no, it's not possible to apply @ApiPageable directly to the parameter instead of the method, but from my side, I don't see that this is a bad idea.

Reply:
> > You added @ApiIgnore, does it make sense to add a description for this param in @ApiParam?
> 
> Yes, because `@ApiIgnore` will hide unused statements in swagger for pageable.

So then why do you need this `@ApiParam` if it's anyway will be hidden?

Reply:
I mean that `Pageable` will not be hidden fully. `@ApiIgnore` will hide these unnecessary options in swagger. 
![image](https://user-images.githubusercontent.com/36819489/122241185-05726c80-cecb-11eb-956c-550541d6adc4.png)

Reply:
ok, got it. Thanks for sharing this screenshot!